### PR TITLE
Add Documentation

### LINKED ISSUE #2: Installation
Hey what does this do, is it production ready and if so how do I install it?

Like so:

```js
import vueBugsnag from 'vueBugsnag'
vueBugsnap.install(Vue, bsconfig);
```

### PR DESCRIPTION
**Pull request by Just-Sieb**
### What's New?

I finally got some spare time to add documentation. Fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+``` javascript
+import VueBugsnag from 'vue-bugsnag';
+
+Vue.use( VueBugsnag, {
+    apiKey: '<insert your bugsnag api key here>',
+    metaData: {}, // Any meta data you want to pass to Bugsnag
+} );
+```
+
+Vue-bugsnags adds four functions to the Vue instance.
```
Comment (toddrobinsonce) [inline README.md]: Vue-bugsnags -> Vue-bugsnag

File: README.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+
+ - `$notify`: This is an alias for `bugsnag.notify` from bugsnag-node
+ - `$leaveBreadCrumb`: This is an alias for `bugsnag.leaveBreadcrumb` from bugsnag-node
+ - `$notifyException`: This is an alias for `bugsnag.notifyException` from bugsnag-node
+ - `$bugsnagSetUser`: Call this function with passing in a string to set `bugsnag.user`
```
Comment (toddrobinsonce) [inline README.md]: `Call this function with passing in a string`
Perhaps better as `Call this function with a string`
or 
`Pass a string into this function to set . . . `

File: README.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
 # vue-bugsnag
```
Comment (toddrobinsonce) [inline README.md]: Is this a convenience wrapper, or are you actually doing some work to make those functions available? I think a short blurb at the top explaining the overall purpose of the code a bit more might help people and potentially make your work easier to find. Something like 

> Bugsnag doesn't have a plugin to connect to the vue instance, so you have to manually do x, y, and z, the vue-bugsnag plugin takes care of those connections and exposes the 4 most common bugsnag functions you need to the vue instance.

**Comment by Just-Sieb** [comment]
Closing as Bugsnag has first class support for Vue now. People should use that.

**Pull request closed without merging**